### PR TITLE
Support 64-bit ints encoded as json strings

### PR DESCRIPTION
**Pull request by davcamer**
The proto3 standard says that both 32-bit and 64-bit integers should be
accepted as json strings. When they are emitted the 32-bit ones should
be json numbers, and the 64-bit json strings.
https://developers.google.com/protocol-buffers/docs/proto3#json

That's what I've aimed to do here. The wrapped int types should follow
the same rules.

### CODE REVIEW

File: src/Protobuf.elm [hunk 1]
```diff
@@ -175,11 +180,36 @@ timestampEncoder v =
...
+
 {-| Decodes an IntValue.
 -}
 intValueDecoder : JD.Decoder Int
 intValueDecoder =
-    JD.int
+    intDecoder
 
 
 {-| Encodes an IntValue.
```
Comment (tiziano88) [inline src/Protobuf.elm:215]: IIRC this is currently used for both 32 and 64 bit wrapper types, ideally we should do the same thing as for the basic types, encoding 32 bit as numbers and 64 bit as strings. WDYT?

Comment (tiziano88) [reply src/Protobuf.elm:215]: Ah just noticed your comment in the conversation above mentioning this :) I'll go ahead and merge your current PR, let me know if you can also look into that, if not I will do it at some point. Thanks a lot!

Comment (davcamer) [reply src/Protobuf.elm:215]: I didn't test this as extensively, but I think this functionality is already there. I was worried how clear it was, and I guess it isn't that clear.

I believe these changes in the go code will mean that wrapped 64 bit Ints will be emitted as strings: https://github.com/tiziano88/elm-protobuf/pull/12/files#diff-346e08d50aa913dadc9d982df75f248aL56

But, it is not symmetric between the 64 and 32 bit wrapped types. The 64 bit types have their encoders generated directly to `numericStringEncoder` but the 32 bit types go indirectly through intValueEncoder. I could tidy that up. Or, maybe I've missed something and the 64 bit wrapped types are broken at the moment?

Comment (davcamer) [reply src/Protobuf.elm:215]: The effect is visible in the generated test files:
https://github.com/tiziano88/elm-protobuf/pull/12/files#diff-6fe72b122f6869f6230aa8d593e66f27L47

**Pull request merged**